.eslintrc.json

```diff
@@ -0,0 +1,48 @@
+{
+  "extends": [
+    "react-app",
+    "airbnb",
+    "plugin:jest/all",
+    "plugin:jsx-a11y/recommended"
+  ],
+  "plugins": [
+    "jest",
+    "jsx-a11y",
+    "react-hooks"
+  ],
+  "rules": {
```

Comment:
Hier überschreiben wir ein paar von den Regeln, die wir oben eingebunden haben, entweder weil sie zu streng sind oder nicht genau passen. Kurze Erklärung dazu:

- `import/extensions`: Das ist standardmäßig nur für `js` und `jsx` auf `"never"`. Da wir aber Typescript verwenden, müssen wir auch noch `ts` und `tsx` hinzufügen.
- `jest/lowercase-name`: Die behalten wir, aber kicken die Regel für `describe`. Nicht so spannend, erkläre ich ggf. in Person, hier etwas schwierig.
- `jest/prefer-expect-assertions`: Finde ich persönlich eine grausame Regel, darum habe ich die ausgestellt. Kann ich bei Bedarf auch noch erklären :grimacing: 
- `react-jsx-filename-extension`: Analog zu oben brauchen wir hier auch noch `.ts` und `.tsx`.